backend/icpy/agent/core/llm/moonshot_client.py

```
@@ -0,0 +1,26 @@
+from ...clients import get_moonshot_client
+
+
+class MoonshotClientAdapter(BaseLLMClient):
+    """Adapter for Moonshot (Kimi) OpenAI-compatible API."""
+
+    def stream_chat(
+        self,
+        *,
+        model: str,
+        messages: List[Dict[str, Any]],
+        tools: Optional[List[Dict[str, Any]]] = None,
```

Ruff (0.14.3)

Unused method argument: `tools`

(ARG002)